common/src/main/java/org/fao/geonet/utils/Xml.java

```diff
@@ -486,6 +486,7 @@ protected static Path resolvePath(Source s) throws URISyntaxException {
                     }
                 }
                 t.transform(srcXml, result);
+                in.close();
```

Comment:
This is not needed since resource `in` is declared in the related `try` using the _[try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)_ and implements the `Closeable` interface.

Reply:
You are ritght !